integration_tests/test_config.ini

```diff
@@ -0,0 +1,2 @@
+# Append the below section(s) to your ~/.jccli.ini in order to run integration tests (see README).
```

Comment:
you're using api key for testing so do you still need this?

Reply:
You are right, we don't, and I just deleted it